src/evict/evict_lru.c

```diff
@@ -147,36 +148,31 @@ __evict_list_clear(WT_SESSION_IMPL *session, WT_EVICT_ENTRY *e)
 {
     if (e->ref != NULL) {
         WT_ASSERT(session, F_ISSET_ATOMIC_16(e->ref->page, WT_PAGE_EVICT_LRU));
-        F_CLR_ATOMIC_16(e->ref->page, WT_PAGE_EVICT_LRU);
+        F_CLR_ATOMIC_16(e->ref->page, WT_PAGE_EVICT_LRU | WT_PAGE_IN_URGENT_QUEUE);
```

Comment:
Would that be ok to rename `WT_PAGE_IN_URGENT_QUEUE` to `WT_PAGE_EVICT_LRU_URGENT` or `WT_PAGE_URGENT_EVICT_LRU`? It would be great to keep a similar name as the existing flag.

Reply:
Updated in [de67431](https://github.com/wiredtiger/wiredtiger/pull/8950/commits/de67431b743a495ffb18d3a0d8900ee33f04a66e)